app/Http/Controllers/TalksController.php

```diff
@@ -25,19 +25,21 @@ class TalksController extends BaseController
         'slides.url' => 'Slides URL must contain a valid URL',
     ];
 
-    protected $sorting_talks = [
+    protected $sorting_style = [
         'date' => '',
         'alpha' => '',
     ];
 
```

Comment:
I like that you cleaned it up--it's definitely better than having style in a PHP var--but we still have the same problem of having style information in our PHP, which is not preferable. If we use a boolean, yes, it clutters up our views, but it makes this array a representation of "which is being sorted by" instead of "classes to apply", which is a much better separation.

To be honest.. one of the interesting questions is, why is this even an array? It feels like there could just be a "sortingBy" property which is set to the name of the field we're sorting by, and then just call it a day.